projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/style.scss

```diff
@@ -0,0 +1,41 @@
+@use "sass:math";
+
+$bar_height: 24px;
+$border_radius: math.div($bar-height, 2);
+
+.mj-boost-speed-score__grade {
+    font-size: 14px;
```

Comment:
I think the `--font-body-small` variable can be used here

Reply:
Ahh, yes thanks! 🙏